### ISSUE TITLE
[Bug]: Can't communicate with devices since 1.28.0-1

### ISSUE THREAD

**Issue by sandert98** [zigbee2mqtt/hassio-zigbee2mqtt#392]
### Description of the issue

Last week, I've set up the Zigbee2Mqtt addon in HomeAssistant OS. Everything was working perfectly, until I came back from a weekend off. Everyting that is connected through Z2M isn't working anymore, and I see a lot of errors in the Zigbee2mqtt logs. Auto update in the addon is turned on, and the only thing I can imagine is the update that is installed this weekend that broke my config. 

Config: 
zigbee_herdsman_debug turned on
Network port 8485
```
enabled: false
master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
options: "-d -d"
log: false`
```
```
port: /dev/ttyACM0
adapter: deconz
```
```
test -w /dev/ttyACM0 && echo success || echo failure
success
```

### Addon version

v1.28.0-1

### Platform

Home Assistant 2022.9.7 
Supervisor 2022.09.1
Operating System 9.0 
Frontend 20220907.2 - latest 

### Logs of the issue (if applicable)

[logs z2m.txt](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/files/9708726/logs.z2m.txt)

**Comment by sandert98** [comment]
For bugfixing, i tried a few things. 
1. Reinstalling the addon. Didn't solve a thing. 
2. Deleting all the devices and connect them again. That did work thankfully. But, there are still some weird things happening: When rebooting the addon, the 'groups' section is corrupt. I get errors that all some lights can't be added to the group, and the group isn't showing up in Home Assistant at all. Deleting the group and recreate it with the lights solves the problem. But, I really don't want to reboot the addon for this reason... Also, when checking for OTA updates, there are a lot of errors of time-outs.

**Comment by nashaden** [comment]
Similar problem here. After updating to 1.28.0-1 all my ZigBee routers are marked as offline. When I restart the addon they appear again and everything is fine for a few hours, after which they stop working again. I reverted to 1.27.2-1 and the problem was gone. Today I updated to 1.28.0-1 and after 4 hours the routers went offline again (restarting the addon made them come back). I reverted to 1.27.2-1 again and am waiting for a bug fix in 1.28.

My routers are (among others) sonoff zbmini, sonoff basic zb3, tuya ts0002_switch_module and many more.

My coordinator is Sonoff ZBBridge with Tasmota 12.1.1.
